presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -42,6 +42,34 @@ function getHumanReadableStateFromInfo(query) {
     );
 }
 
+
+function ResourceGroupLinks({groupId, length=35}) {
+    if (groupId && groupId.length > 0) {
```

Comment:
I think we can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and implement the breaker pattern.

Adding a breaker is something like this, it is the first thing to do in the component. The goal is to remove the clutter that can cause many `if` statements and focus the react component to what it really does.

```
if(!groupId?.length) return ('n/a');
```

Reply:
fixed